is_prime.py

```diff
@@ -0,0 +1,9 @@
+def is_prime(n):
```

Comment:
Again, pretty minor, but it would be nice to get into a habit of avoiding non-descriptive variable names like `n`. Maybe use `num` or `number` here? With any decent IDE, I find using a more verbose name to help with auto-completion, and make the code read better.